File: 250228/두수의 곱과 몫/multiple-and-quotient-of-two-numbers.py
user = input()
arr = user.split()

a = int(arr[0])
b = int(arr[1])
c = a * b
d = int(a / b)

if 1 <= b <= a <= 500 :
    print(f"{a} * {b} = {c}")
    print(f"{a} / {b} = {d}")
else :
    print("error")

File: 250721/두수의 곱과 몫/multiple-and-quotient-of-two-numbers.swift
import Foundation

let input = readLine()!.split(separator: " ").map { Int(String($0))! }
let a = input[0]
let b = input[1]

if (1 <= b && b <= a && a <= 500) {
    print("\(a) * \(b) = \(a * b)")
    print("\(a) / \(b) = \(a / b)")
}